docs/AboutUs.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: About Us
+---
+
+AddressBook - Level 3 was developed by the
+[se-edu](https://se-edu.github.io/docs/Team.html) team.
+*{The dummy content given below serves as a placeholder to be used by
+future forks of the project.}*
+
+We are a team based in the [School of Computing, National University of
+Singapore](http://www.comp.nus.edu.sg).
+
+## Project Team
+
+### John Doe
+
+![damithc](images/damithc.jpg)
+
+\[[homepage](http://www.comp.nus.edu.sg/~damithch)\]
+\[[github](https://github.com/damithc)\]
+\[[portfolio](team/johndoe.md)\]
+
+Role: Project Advisor
+
+### John Roe
+
+![lejolly](images/lejolly.jpg)
```

Comment:
Some of these images are too big. use `<img ... width ../>`?

Reply:
Resized everything uniformly.